src/fsharp/vs/service.fs

```diff
@@ -2589,31 +2589,37 @@ type BackgroundCompiler(referenceResolver, projectCacheSize, keepAssemblyContent
           })
             
     member bc.InvalidateConfiguration(options : FSharpProjectOptions) =
+        // This operation can't currently be cancelled nor awaited
```

Comment:
Invalidate configuration is adjusted to explicitly invalidate any "related" project